apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsLinkControl.tsx

```diff
@@ -89,24 +205,78 @@ const RadioCard = ({ children, ...rest }: PropsWithChildren<UseRadioProps>) => {
 
 interface RadioContentProps {
   selectedLinkType: string
+  description?: string
   data: string
   handleChange: (value: string) => void
 }
 
 const RadioContent = ({
   selectedLinkType,
+  description,
   data,
   handleChange,
 }: RadioContentProps): JSX.Element => {
+  const router = useRouter()
+  const siteId = String(router.query.siteId)
```

Comment:
I'd suggest against coupling this component with query params! Should we pass in the siteId as a prop instead?

Reply:
Annoyingly this is a JSONForms component so we can't modify the props that we pass in here (since we can't specify the kind of props to pass to this component as it is entirely managed by JSONForms), hence opted to retrieve this information using the query params.